website/orders/templates/orders/shift_overview.html

```diff
@@ -1,42 +1,42 @@
-{% extends 'orders/order_base.html' %}
-{% load static %}
+{% extends 'tosti/base.html' %}
+{% load static order_now %}
 
 {% block styles %}
     <link rel="stylesheet" href="{% static 'tosti/css/index.css' %}"/>
     <link rel="stylesheet" href="{% static 'orders/css/forms.css' %}"/>
-    <link rel="stylesheet" href="{% static 'orders/css/item-list-admin.css' %}"/>
 {% endblock %}
 
-{% block order_content %}
-    {% if user.is_staff %}
-        <div class="container-sm">
-            <div class="btn-ml btn-on bg-primary border-0 text-center" onclick="window.location.href = '{% url "orders:shift_admin" shift=shift %}';">
-                <h5>Admin view</h5>
-                <i class="fas fa-tools"></i>
+{% block page %}
+    {% render_order_header shift %}
+    {% render_order_items shift user=request.user %}
+{% endblock %}
+
+{% block footer %}
+    {% if user.is_staff and user not in shift.get_assignees %}
+        <footer class="page-footer navbar navbar-expand-md fixed-bottom">
+            <div class="container text-center">
+                <div class="row flex-grow-1">
+                    <div class="col">
+                        <div class="btn-ml m-auto cursor-pointer btn-on" onclick="window.location.href = '{% url 'orders:shift_join' shift=shift %}';">
+                            <i class="fas fa-sign-in-alt"></i>
```

Comment:
I still suggest to change some of these icons to the alternatives I sent you via whatsapp

Reply:
They are still a bit unclear now, and imo a UI should never be explained